src/edu/stanford/dlss/was/WasapiDownloader.java

```diff
@@ -9,12 +9,19 @@
   public WasapiDownloaderSettings settings = null;
 
 
-  public WasapiDownloader(String settingsFileLocation, String[] args) throws IOException {
+  public WasapiDownloader(String settingsFileLocation, String[] args) throws LoadSettingsException {
     settings = new WasapiDownloaderSettings(settingsFileLocation, args);
   }
 
+  public void run() {
```

Comment:
yeah, i had a similar feeling initially and ignored it.  turns out `execute` is used as a method in some Java Swing classes, so i went with the slightly more verbose `executeFromCmdLine`, since the point of that method was to do the stuff you'd do from the command line, while keeping `main` relatively concise.